py/foundation/entity.py

```diff
@@ -105,6 +119,13 @@ def from_proto(msg: PbEntityPayloadType) -> 'LineEntity':
     ocr_words = [OcrWordEntity(InputWord.from_proto(w)) for w in msg.ocr_words]
     return LineEntity(ocr_words)
 
+  def to_proto(self) -> entity_pb2.Line:
+    msg = entity_pb2.Line()
+    for w in self._ocr_words:
+      input_word_msg = w.word.to_proto()
+      msg.ocr_words.append(input_word_msg)
```

Comment:
Does this actually work? I thought that you couldn't append to repeated fields, and that you had to append to a separate list first.

Reply:
I used https://developers.google.com/protocol-buffers/docs/reference/python-generated#repeated-fields as a reference, it seems they append to the field?

Reply:
Can we double check with a test?

Reply:
Just confirmed this works! Also this might be a good time to setup unit testing.